config/vlan.py

```diff
@@ -50,6 +51,34 @@ def del_vlan(db, vid):
         db.cfgdb.set_entry('VLAN_MEMBER', k, None)
     db.cfgdb.set_entry('VLAN', 'Vlan{}'.format(vid), None)
 
+def restart_ndppd():
+    docker_exec_cmd = "docker exec -it swss {}"
```

Comment:
Can you check the behavior when swss docker is not running and user configures proxy_arp?

Reply:
It should just be something like this:
```
Error response from daemon: Container a183b127127877c76698526e4283491f1fb45243d2e9a3ab80368214f597ff25 is not running
```

Is that fine, or should I check if the container is up first?

Reply:
I think yes, if user configures proxy-arp and see this error, it might be confusing whether it got applied.